querybook/webapp/ui/FlowGraph/GraphControls.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {
+    Controls,
+    ReactFlowState,
+    useReactFlow,
+    useStore,
+    useStoreApi,
+} from 'react-flow-renderer';
+
+import { IconButton } from 'ui/Button/IconButton';
+
+import { getLayoutedElements, LayoutDirection } from './helpers';
+
+import './GraphControls.scss';
+
+const isInteractiveSelector = (s: ReactFlowState) =>
+    s.nodesDraggable && s.nodesConnectable && s.elementsSelectable;
+
+export const GraphControls = () => {
+    const store = useStoreApi();
+    const isInteractive = useStore(isInteractiveSelector);
+    const { zoomIn, zoomOut, fitView } = useReactFlow();
+
+    const reactFlowInstance = useReactFlow();
+
+    const onLayout = React.useCallback((direction: LayoutDirection = 'LR') => {
+        const layoutedDAG = getLayoutedElements(
+            reactFlowInstance.getNodes(),
+            reactFlowInstance.getEdges(),
+            direction
+        );
+        reactFlowInstance.setNodes([...layoutedDAG.nodes]);
+        reactFlowInstance.setEdges([...layoutedDAG.edges]);
+    }, []);
+
+    const onZoomIn = () => zoomIn();
```

Comment:
this is same as `const onZoomIn = zoomIn`

Reply:
the parameter type of `zoomIn` doesn't match IconButton's `onClick`

Reply:
ah okay, can you wrap these functions with useCallback as well